### ISSUE TITLE
[UI] Unable to click buttons 'C' and 'A'

### ISSUE THREAD

**Issue by mspykerez** [SimCMinMax/AethysRotation#155]
I'm using the latest alpha and 'O' works to disable/enable the addon, the others C A  don't do anything...

**Comment by Kutikuti** [comment]
Working fine for me. Which rotation are you talking ?

**Comment by mspykerez** [comment]
To me its happening on all specs! But I tested it further, and find out that unlocking the addon it lets me move the buttons alone to the right which fixes it thou it kinda stays out of place regarding the rest of the addon icons. If I move the buttons to the left I am unable to click all 3 buttons and it don't even let me move them afterwards, I had to reset the position of the buttons with the 'reset buttons' option to be able to move them again.
